src/order/filter/countable_Inter.lean

```diff
@@ -63,3 +63,27 @@ by { rw ← principal_empty, apply countable_Inter_filter_principal }
 
 instance countable_Inter_filter_top : countable_Inter_filter (⊤ : filter α) :=
 by { rw ← principal_univ, apply countable_Inter_filter_principal }
+
+/-- Infimum of two `countable_Inter_filter`s is a `countable_Inter_filter`. This is useful, e.g.,
+to automatically get an instance for `residual α ⊓ principal s`. -/
+instance countable_Inter_filter_inf {l₁ l₂ : filter α} [countable_Inter_filter l₁]
```

Comment:
Could you make the arguments explicit? Normally, it is an instance so it should never be called, but if one day someone needs to call it then explicit arguments seem better. Same thing for the next instance.

Reply:
```suggestion
instance countable_Inter_filter_inf (l₁ l₂ : filter α) [countable_Inter_filter l₁]
```